backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/logging/impl/Log.java

```diff
@@ -80,8 +81,8 @@
 			value = "Elasticsearch request failed: %3$s\nRequest: %1$s\nResponse: %2$s"
 	)
 	SearchException elasticsearchRequestFailed(
-			@FormatWith( ElasticsearchRequestFormatter.class ) ElasticsearchRequest request,
-			@FormatWith( ElasticsearchResponseFormatter.class ) ElasticsearchResponse response,
+			@FormatWith(ElasticsearchRequestFormatter.class) ElasticsearchRequest request,
```

Comment:
I agree these changes are correct, but would you mind moving them to a separate commit dedicated to formatting changes?

Reply:
i tried to remove my first commit but it didn't workout ^^